source/features/bugs-tab.tsx

```diff
@@ -24,16 +24,40 @@ async function highlightBugsTabOnIssuePage(): Promise<void | false> {
 	issuesTab.removeAttribute('aria-current');
 }
 
+const cacheKey = (): string => 'bugs-label' + ':' + getRepo()!.nameWithOwner;
 const countBugs = cache.function(async (): Promise<number> => {
-	const {search} = await api.v4(`
-		search(type: ISSUE, query: "label:bug is:open is:issue repo:${getRepo()!.nameWithOwner}") {
-			issueCount
+	if (!await cache.get(cacheKey())) {
+		const {repository} = await api.v4(`
+			repository() {
+				labels(query: "bug", first: 10) {
+					nodes {
+						name
+						issues(states: OPEN) {
+							totalCount
+						}
+					}
+				}
+			}
+		`);
+
+		const {name: bugLabel, issues} = [...repository.labels.nodes].find(label => [':bug: bug', 'bug', 'confirmed-bug', 'type: bug'].includes(label.name.toLowerCase()));
+		void cache.set(cacheKey(), bugLabel.includes(':') ? `"${bugLabel}"` : bugLabel ?? false);
```

Comment:
_I know this is a draft PR_ 😁 

Let's always wrap it in quotes, but not here, we should store the pure label name so it can be potentially reused in different contexts (where `"` don't apply, for example)

Also I think `?? false` is mispositioned here? If it's not found, `find` will return `undefined` and the destructuring on the same line will fail.

Reply:
I know 👍.

The issue is that when searching for a bug page it will only match when using RG bugs-tab.

So we need to match GitHub search.

The code needs a real cleanup, so I will work on it.